VelociraptorPlatform-Electron/public/lib/terminal-emulator.js

```
@@ -0,0 +1,374 @@
+        _updateSearchCount() { this.elements.searchCount.textContent = `${this.searchMatches.length ? this.searchIndex + 1 : 0}/${this.searchMatches.length}`; }
+
+        _renderOutput() {
+            const tab = this._getActiveTab(); if (!tab) return;
+            this.elements.output.innerHTML = tab.output.map(item => `<div class="terminal-line ${item.type}-line">${this._parseAnsi(escapeHtml(item.content))}</div>`).join('');
+            this._scrollToBottom();
+        }
+
+        _parseAnsi(text) {
+            const map = { '0': '</span>', '1': '<span class="ansi-bold">', '2': '<span class="ansi-dim">', '31': '<span class="ansi-red">', '32': '<span class="ansi-green">', '33': '<span class="ansi-yellow">', '34': '<span class="ansi-blue">', '35': '<span class="ansi-magenta">', '36': '<span class="ansi-cyan">', '37': '<span class="ansi-white">' };
+            // eslint-disable-next-line no-control-regex
+            return text.replace(/\x1b\[(\d+)m/g, (_, c) => map[c] || '');
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)